prod-devel-rcar-s4.yaml

```diff
@@ -171,12 +174,10 @@ components:
         - "../meta-openembedded/meta-oe"
         - "../meta-openembedded/meta-filesystems"
         - "../meta-openembedded/meta-python"
-        - "../meta-renesas/meta-rcar-gen3"
```

Comment:
Do you plan to delete it at all?

Reply:
This is Renesas Gen3 meta-layer. It was used as an BSP before we get dedicated layer for Gen4. What do you mean by "delete it"?

Reply:
oh, that one os wrong